docs/guides/ha/cluster-setup.md

```
@@ -0,0 +1,388 @@
+|------|------------|--------------|----------|
+| node-1 | 10.0.0.1 | 10.0.0.1:5001 | 26656 |
+| node-2 | 10.0.0.2 | 10.0.0.2:5001 | 26656 |
+| node-3 | 10.0.0.3 | 10.0.0.3:5001 | 26656 |
+| node-4 | 10.0.0.4 | 10.0.0.4:5001 | 26656 |
+| node-5 | 10.0.0.5 | 10.0.0.5:5001 | 26656 |
+
+Replace these with your actual IP addresses throughout the guide.
+
+P2P peers use the libp2p multiaddr format, which includes each node's peer ID:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)